content/en/docs/instrumentation/erlang/resources.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Resources
+weight: 36
+---
+
+## Resources
+
+A [resource](/docs/reference/specification/overview/#resources) represents the
+entity producing telemetry as resource attributes. For example, an OTP Release
+producing telemetry that is running in a container on Kubernetes has an OTP
+Release name, a Pod name, a namespace, and possibly a deployment name. All four
+of these attributes can be included in the resource.
+
+In your observability backend, you can use resource information to better
+investigate interesting behavior. For example, if your trace or metrics data
+indicate latency in your system, you can narrow it down to a specific container,
+pod, or Kubernetes deployment.
+
+### Using resource detectors
+
+Resource detectors fetch resource attributes from various sources. The default
+detectors use the OS environment variable `OTEL_RESOURCE_ATTRIBUTES` and the
+`opentelemetry` OTP Application environment variable `resource`.
```

Comment:
I'm confused here. This defines the resource stuff, but we then introduce the resource detectors, and only later come back to resources.

I think this first paragraph needs reworking to either go directly to resource detectors, or to properly describe that we'll see both the detectors and the resource declarations. A third option would be to introduce both at once I guess, if they're not really independent.

Reply:
Yea.. I actually had them flipped originally. Since adding resource attributes requires detectors I thought just covering detectors first made sense (since the first paragraph covers what these things are).

But the user certainly doesn't need to understand, or even know about, detectors to make use of adding resource attributes, so I can see why it would make sense to flip them back.